crates/aptos/CHANGELOG.md

```diff
@@ -3,7 +3,10 @@
 All notable changes to the Aptos CLI will be captured in this file. This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and the format set out by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+
+## [3.3.0] - 2024/05/03
 - **Breaking Change** Update View functions to use BCS for submission.  Allows for all arguments to be supported in view functions.  Note some input arguments that were previously inputted as strings may be handled differently.
+- Early beta release of the Move compiler v2 is now accessible through the CLI. We now allow specifying the Move compiler version and the Move language version via the CLI.
```

Comment:
I wonder whether we can link to the doc page once that has landed. We can coordinate quickly on doing it shortly after each other.

Reply:
Sounds good.

Reply:
Done.